docs/orchestrating-elastic-stack-applications/agent.asciidoc

```diff
@@ -255,6 +255,10 @@ spec:
 ...
 ----
 
+[id="{p}-elastic-agent-connect-es"]
+=== Customize the connection to an Elasticsearch cluster
+
+The `elasticsearchRefs` element allows ECK to automatically configure Elastic Agent to establish a secured connection to one or more managed Elasticsearch clusters. By default it targets all nodes in your cluster. If you want to direct traffic to specific nodes of your Elasticsearch cluster see  <<{p}-traffic-splitting>> for more information and examples.
```

Comment:
```suggestion
The `elasticsearchRefs` element allows ECK to automatically configure Elastic Agent to establish a secured connection to one or more managed Elasticsearch clusters. By default it targets all nodes in your cluster. If you want to direct traffic to specific nodes of your Elasticsearch cluster, refer to  <<{p}-traffic-splitting>> for more information and examples.
```
To meet accessibility guidelines, we should avoid terms like "see". Refer, check are some alternatives.